### ISSUE TITLE
API object detection num_epochs parameters

### ISSUE THREAD

**Issue by Shiro-LK** [tensorflow/models#4368]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10 - anaconda
- **TensorFlow version (use command below)**: 1.8
- **Exact command to reproduce**: `python models\research\object_detection\train.py --logtostderr --train_dir=API_tf\training\ --pipeline_config_path=API_tf\model\pipeline_ssdlite.config`

### Describe the problem
I choose to set the num_epochs value to 10 (input_reader training parameters) inside the config file and the number of steps equal to 5. But when I launched the training, it stopped after 5 steps. So I suppose that, the num_epochs parameters doesn't work (or maybe I misunderstanding how it works?)

### Source code / logs

> INFO:tensorflow:Restoring parameters from API_tf\model\model.ckpt
> INFO:tensorflow:Running local_init_op.
> INFO:tensorflow:Done running local_init_op.
> INFO:tensorflow:Starting Session.
> INFO:tensorflow:Saving checkpoint to path API_tf\training\model.ckpt
> INFO:tensorflow:Starting Queues.
> INFO:tensorflow:global_step/sec: 0
> INFO:tensorflow:Recording summary at step 0.
> INFO:tensorflow:global step 1: loss = 10.5835 (58.332 sec/step)
> INFO:tensorflow:global step 2: loss = 10.1586 (21.762 sec/step)
> INFO:tensorflow:global step 3: loss = 9.6173 (23.551 sec/step)
> INFO:tensorflow:global_step/sec: 0.0257188
> INFO:tensorflow:global step 4: loss = 9.1852 (31.616 sec/step)
> INFO:tensorflow:Recording summary at step 4.
> INFO:tensorflow:global step 5: loss = 9.2487 (23.212 sec/step)
> INFO:tensorflow:Stopping Training.
> INFO:tensorflow:Finished training! Saving model to disk.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
TensorFlow installed from
Bazel version
CUDA/cuDNN version
GPU model and memory

**Comment by Shiro-LK** [comment]
Thank you for the answer, I edited my first post.

**Comment by Shiro-LK** [comment]
I see, thank for the answer !

**Comment by AswathKiruba** [comment]
@pkulzc Thanks for the answer. When I was running python eval.py --logtostderr  --checkpoint_dir=path/to/checkpoint_dir  --eval_dir=path/to/eval_dir --pipeline_config_path=pipeline_config.pbtxt

I am getting the results (prediction, images with bounding boxes) on the tensorboard. But the result images tend to be jumbled when compared to the original test images. Do you know why this occurs?

**Comment by QuickLearner171998** [comment]
I have 500 images in my training data and I set batch size = 5 and num_steps = 20k. Does that mean number of epochs are equal to 200 ??

**Comment by Annieliaquat** [comment]
> I have 500 images in my training data and I set batch size = 5 and num_steps = 20k. Does that mean number of epochs are equal to 200 ??

do you know now that how to calculate num of epochs
